packages/fxa-shared/metrics/glean/fxa-backend-metrics.yaml

```diff
@@ -720,6 +720,29 @@ third_party_auth:
         description: |
           Indicates the initial linking of the Mozilla account and the third-party account.
         type: boolean
+  apple_login_complete:
+    type: event
+    description: |
+      A user successfully logged in with Apple OAuth.
```

Comment:
nit: Not sure how much description needs to go into these, but it might be worthwhile to mention that the Mozilla account was also created if it didn't already exist.